fixtures/room/daily_room_events_fixture.py

```diff
@@ -40,8 +47,54 @@
                     'startTime': '09:00:00'
                 }
             ]
-        }]
+        }
+      ]
+    }
+  }
+}
+
+many_daily_room_events_query = '''
```

Comment:
This query name `many_daily_room_events_query` is a bit confusing. Please use one that is more descriptive.

Reply:
@nakatuddesuzan This has been worked on. Thanks for the feedback